Sources/Generics/GenericTableViewController.swift

```diff
@@ -60,59 +61,40 @@ class GenericTableViewController<T: GenericCell<ModelType>, ModelType>: UITableV
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        self.titleDisplayMode(.never)
-        self.extendedLayoutIncludesOpaqueBars = true
+
+        view.backgroundColor = .white
```

Comment:
Do we need this `white` color? If yes then you should use `Theme`.

Reply:
No need to add this color, copied from previous pr